src/docs/Help/WantToHelp/index.mdx

```diff
@@ -1,21 +1,15 @@
 ---
```

Comment:
Hi @caqueste , Hi @mohamedMok ,
I wonder if this page should not be deleted altogether? Indeed, it is updated very rarely, is it useful to keep it?

Reply:
@mohamedMok @tiloyi I agree, we can delete this page, I think it's useless 😉